src/node_crypto.cc

```diff
@@ -20,6926 +20,20 @@
 // USE OR OTHER DEALINGS IN THE SOFTWARE.
 
 #include "node_crypto.h"
-#include "node_buffer.h"
```

Comment:
It’s a bit unfortunate that so much of the splitting is still done in this commit … that definitely gets in the way a bit with reviewing, and I’m also definitely concerned about backporting this – the current state would essentially make every crypto change require a manual backport to all release branches that don’t have webcrypto

Reply:
Once this lands, I'll be working on a set of backports that limit the possibly breaking changes to the error messages) and will likely initially omit the webcrypto api bits.